emscripten/server.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+var repl = require('repl');
+var mpy = require('./main.js');
```

Comment:
Since it's now importable, this line can be `var mpy = require('.');` instead, that's cleaner.